deploy/Dockerfile

```diff
@@ -5,9 +5,9 @@ WORKDIR /src/github.com/AliyunContainerService/kube-eventer
 RUN apt-get update -y && apt-get install gcc ca-certificates
 RUN make
 
-FROM alpine:3.10
+FROM golang:1.12.6 
```

Comment:
Why change alpine:3.10 to golang:1.12.6

Reply:
Sorry, that was an accident, I'll remove it.

Reply:
Thanks